pusher/src/Services/SocketManager.ts

```diff
@@ -121,28 +123,15 @@ export class SocketManager implements ZoneEventListener {
                 }
             })
             .on("end", () => {
-                console.warn(
-                    "Admin connection lost to back server '" +
-                        apiClient.getChannel().getTarget() +
-                        "' for room '" +
-                        roomId +
-                        "'"
-                );
+                console.warn("Admin connection lost to back server");
                 // Let's close the front connection if the back connection is closed. This way, we can retry connecting from the start.
                 if (!client.disconnecting) {
                     this.closeWebsocketConnection(client, 1011, "Admin Connection lost to back server");
                 }
                 console.log("A user left");
             })
             .on("error", (err: Error) => {
-                console.error(
-                    "Error in connection to back server '" +
-                        apiClient.getChannel().getTarget() +
-                        "' for room '" +
-                        roomId +
-                        "':",
-                    err
-                );
+                console.error("Error in connection to back server:", err);
```

Comment:
You are removing a commit I made to add more details here. Are you sure you did not force push something not intended?